src/tokens/types.ts

```diff
@@ -1,15 +1,16 @@
 import Colors from 'src/styles/colors'
 
-export enum TokenDetailsActionName {
+export enum TokenActionName {
   Send = 'Send',
   Swap = 'Swap',
   Add = 'Add',
   Withdraw = 'Withdraw',
   More = 'More',
+  Receive = 'Receive',
```

Comment:
should this just be `Transfer` everywhere to match with the name in the designs?